ee/ucp/deploy-application-package.md

```diff
@@ -35,19 +35,19 @@ To create a stack in the UCP web interface, follow these steps:
 
 6. Select either **Compose File** or **App Package** for the **Application File Mode**.
 7. Select **Next**.
```

Comment:
Change to read, "Click **Next** to activate **Section 2. Add Application File**."